src/mmw/js/src/core/tests.js

```diff
@@ -30,51 +30,55 @@ var chartData = [{x: 'a', y: 1},
                 {x: 'c', y: 3}],
     xValue = 'x',
     yValue = 'y',
-    sandboxHeight = '500',
-    sandboxWidth = '700',
-    sandboxSelector = '#display-sandbox';
+    displaySandboxHeight = '500',
+    displaySandboxWidth = '700',
+    displaySandboxSelector = '#display-sandbox',
+    sandboxId = 'sandbox',
+    sandboxSelector = '#' + sandboxId;
```

Comment:
I think this is a good idea too. Also, this makes me think we might want to create a test bootstrap file which could also set up the app reset method.